.github/workflows/foxy-binary-build.yml

```diff
@@ -0,0 +1,61 @@
+name: Foxy Binary Build
+on:
+  push:
+    branches:
```

Comment:
Maybe remove this here and only leave it on the foxy branch - similar to the galactic solution?